.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
Why are there so many node versions?

Reply:
I think this is just done to ensure that the build supports all the different node versions.
It is part of the update from https://github.com/probot/stale

Reply:
Why not just the latest?

Reply:
Actually, I am fine with not having ci tests on this for repo for now since we are anyways planning to deprecate it.

Reply:
Okay, I'd be removing this